src/client/containers/PullRequestDiff/__tests__/index.js

```diff
@@ -0,0 +1,69 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
+  it('full data', () => {
```

Comment:
There is `fetchStatus` selector that might not find any status by `FETCH_PULL_REQUEST_DIFF` action type(let's assume this). Would be great to make a test for this case and expect it returns `null` or empty object - whatever it now returns. UI component will on this result of course. In the future, if someone changes something in fetch status selectors logic and it, for example, will start to return different result for non-existent field or throw an exception - we've got our unit test failed and can fix UI component accordingly.